k8-util/helm/fluvio-app/templates/fluvio-connector-config.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: fluvio-config-map
+data:
+  fluvioClientConfig: |
+    version = "2.0"
+    current_profile = "k8-config-map-deployment"
+
+    [profile.k8-config-map-deployment]
+    cluster = "k8-config-map-deployment"
+
+    [cluster.k8-config-map-deployment]
+    {{ if .Values.tls }}
```

Comment:
I've tested this with `smoke-test-k8-tls` and without to verify the different endpoints are applied.

Reply:
So we have port number hard coded.  Not show stopper for now.